segmenters/nlp/JiebaSegmenter/__init__.py

```diff
@@ -40,6 +53,10 @@ def segment(self, text: str, *args, **kwargs) -> List[Dict]:
         :rtype: List[Dict]
         """
         import jieba
+
+        if self.user_dict_file is not None:
```

Comment:
This part and the checking of the existance of user dict file should be handled at `post_init()` method.

Post init is a special method in Jina where dependencies are loaded and expensive setting is done. It is our common pattern to work like this.

This way you do not need to load the user dict for every segment call

Reply:
ah that sounds better indeed, thx.
I am using post_init now and also call super there as I saw it in other places, but I am not 100% sure if that is correct here. I suppose so?

Reply:
Yes, it is great!